src/util/std_code.h

```diff
@@ -105,9 +105,21 @@ class code_blockt:public codet
   {
   }
 
+  typedef std::vector<codet> code_operandst;
+
+  code_operandst &statements()
+  {
+    return (code_operandst &)get_sub();
```

Comment:
Use `static_cast`

Reply:
That doesn't work, the C++ type system isn't that elaborate.